src/cml.js

```diff
@@ -143,59 +143,57 @@ class CML {
     return await getDriver(this).runnerToken();
   }
 
-  parseRunnerLog(opts = {}) {
+  async parseRunnerLog(opts = {}) {
```

Comment:
Unless I miss something, adding the `async` keyword here would require the use of `await` when calling this function, right?

https://github.com/iterative/cml/blob/c51e8706d60246b8a1920a6c4a441919306539e4/bin/cml-runner.js#L228